### DISCUSSION TITLE
Improve the Population.summary output with a parameter to highlight when the parameter is best

### DISCUSSION THREAD

**Discussion by CarloNicolini** [skfolio/skfolio#23] [Ideas]
## Description
The `Population.summary` method currently includes the `formatted` parameter, which is useful. 

However, to enhance user experience, it would be beneficial to introduce another parameter to show which metric is best over all columns.
This new parameter would facilitate a quick comparison of metrics, allowing users to identify the best or worst performing aspects of the population.

Something similar to the pandas styler `.style.highlight_max` but here should work based not on max or min but on *best* condition.

**Comment by HugoDelatte** [comment]
Yes it's a good idea.
The solution would be to highlight the maximum for `PerfMeasure` and `RatioMeasure` and the minimum for `RiskMeasure` using the same logic implemented [here](https://github.com/skfolio/skfolio/blob/1b312ac8180434a2df1d1e415b7f1be75ce43b02/src/skfolio/metrics/_scorer.py#L116)
